completions/ant

```diff
@@ -9,7 +9,7 @@ _ant_parse_targets()
     # parse buildfile for targets
```

Comment:
```suggestion
    # parse buildfile for targets
    ret=()
```

or if we still use a space-separated result in the scalar `ret`,

```suggestion
    # parse buildfile for targets
    ret=""
```

Reply:
Actually this function is recursive, so initialization needs to happen outside of it. Addressed in the latest commit.

Reply:
Ah, I see. Thank you.